frontend/src/metabase/ui/components/overlays/Popover/Popover.styled.tsx

```diff
@@ -7,6 +7,7 @@ export const getPopoverOverrides = (): MantineThemeOverride["components"] => ({
       shadow: "md",
       withinPortal: true,
       middlewares: { shift: true, flip: true, size: true },
+      transitionProps: { duration: 0 },
```

Comment:
I'm pretty sure we want popover transitions to be consistent and not tweak it every time we use it.

Reply:
I agree.

Why did you decide to revert this change?